src/Api/Serializer/DiscussionSerializer.php

```diff
@@ -37,28 +37,28 @@ protected function getDefaultAttributes($discussion)
         $gate = $this->gate->forUser($this->actor);
 
         $attributes = parent::getDefaultAttributes($discussion) + [
-            'commentsCount'     => (int) $discussion->comments_count,
-            'participantsCount' => (int) $discussion->participants_count,
-            'startTime'         => $this->formatDate($discussion->start_time),
-            'lastTime'          => $this->formatDate($discussion->last_time),
+            'commentsCount'     => (int) $discussion->comment_count,
+            'participantsCount' => (int) $discussion->participant_count,
+            'startTime'         => $this->formatDate($discussion->created_at),
+            'lastTime'          => $this->formatDate($discussion->last_posted_at),
```

Comment:
We should rename the API attributes as well, but understandable leaving that to a separate PR.